domain-cc/cc-api-controller/src/main/java/gov/va/vro/controller/cc/v3/ContentionClassificationController.java

```diff
@@ -0,0 +1,71 @@
+package gov.va.vro.controller.cc.v3;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import gov.va.vro.api.cc.ResourceException;
+import gov.va.vro.api.cc.v3.CCResource;
+import gov.va.vro.api.cc.v3.ResourceResponse;
+import gov.va.vro.camel.CamelEntry;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONObject;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class ContentionClassificationController implements CCResource {
+  private final CamelEntry camelEntry;
+  private final String EXCHANGE_NAME = "contention-classification-exchange";
+  private final String ENDPOINT_NAME = "domain-cc-classify";
+
+  // Get a POJO from the Camel response so jackson can automagically serialize the HTTP response
+  // https://stackoverflow.com/a/44842806
+  // https://stackoverflow.com/a/45465724
+  public HashMap<String, Object> getMapFromString(String jsonString) {
+    final ObjectMapper mapper = new ObjectMapper();
+    HashMap<String, Object> mapFromString = new HashMap<>();
+    try {
+      mapFromString = mapper.readValue(jsonString, new TypeReference<HashMap<String, Object>>() {
+      });
+    } catch (IOException e) {
+      log.error("Exception launched while trying to parse String to Map.", e);
+    }
+    return mapFromString;
+  }
+
+  @Override
+  public ResponseEntity<ResourceResponse> callEndpoint(String endpoint, JsonNode request)
```

Comment:
Having a `JsonNode` as a parameter is curious -- I'll need to look into this since I was expecting a custom object like https://github.com/department-of-veterans-affairs/abd-vro/blob/39a1d2ef921d05ad93132487fb487a5b597a3947/domain-xample/xample-api-controller/src/main/java/gov/va/vro/controller/xample/v3/XampleController.java#L31

Reply:
My thinking was that the url of the POST specifies what endpoint needs accessed in Python
`/v3/domain-cc/{endpoint}` e.g. `endpoint=get_classification` = `/v3/domain-cc/get_classification`
And the entire JSON payload is sent to that endpoint. 

That way, FastAPI's Pydantic models can do the validation. [Here is an example](https://fastapi.tiangolo.com/tutorial/body-nested-models/#list-fields) of an "Item" that must be sent to the endpoint. We can define our own pydantic models and endpoints in Python w/o having to generate a new java class for ea one.

If there's an issue w/ processing the request, whatever status code and error message comes from the underlying python api gets forwarded to the VRO api consumer.